UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -1773,5 +1783,15 @@ public void TestInterfaceObjectMarshalling()
 
             Assert.Equal(TestObject.ReadWriteProperty, nativeProperties.ReadWriteProperty);
         }
+
+        [Fact]
```

Comment:
what is this testing, related to generic MarshalInspectable?  can you add a comment?